sys/net/gnrc/network_layer/ipv6/Makefile

```diff
@@ -1,3 +1,7 @@
 MODULE = gnrc_ipv6
 
+ifeq (,$(filter FEATURES_PROVIDED,periph_gpio))
+  # this should not be
+endif
```

Comment:
These are reported in the `static-tests` output as annotations, but there seems to be a cap of 10 per check, for if they are reported in the summary or in the file.